ehitchhiking-ui/src/app/shared/components/trip/trip.component.html

```diff
@@ -0,0 +1,38 @@
+<div class="trips" >
+<div
+  class="trips-user"
+>
+  <i class="material-icons">
+    directions_car
+  </i>
+  <p class="trips-list truncated">{{trip.startPoint}}  ->  {{trip.endPoint}}</p>
+  <div class = "form-row">
+    <button class = "trip-btn" >
+      <i class="material-icons" *ngIf="trip.status === 0" title="active trip">
+        event_note
+      </i>
+      <i class="material-icons" *ngIf="trip.status === 1" title = "completed trip">
+        event_available
+      </i>
+      <i class="material-icons" *ngIf="trip.status === 2" title = "declined trip">
+        event_busy
+      </i>
+    </button>
+    <button class = "trip-btn" title = "rating">
+      <i class="material-icons">
+        star_border
+      </i>
+    </button>
+    <button class = "trip-btn"
+            [class.active]="trip.isFavorite"
+            (click)="handleChange('favorite')">
+      <i class="material-icons" *ngIf="!trip.isFavorite" title = " no favorite trip">
```

Comment:
there is no need to display two buttons. Pls use ng-class to get special css classes for favorite mode + dynamic title